src/main/java/mate/academy/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file has wrong location. It is not java
![image](https://github.com/mate-academy/jv-logger/assets/20326858/03c152b8-633c-41aa-9e34-8a7101041a36)